backend/tournesol/admin.py

```diff
@@ -9,15 +9,15 @@
     ComparisonCriteriaScore,
     ContributorRating,
     ContributorRatingCriteriaScore,
-    Video,
-    VideoCriteriaScore,
+    Entity,
+    EntityCriteriaScore,
 )
 
 
-@admin.register(Video)
+@admin.register(Entity)
 class VideoAdmin(admin.ModelAdmin):
```

Comment:
Looks like we did not anticipate the need to rename the `VideoAdmin` and `VideoCriteriaScoreAdmin` class? 

I'd recommend renaming it right away, because it is easy and it would be very confusing for someone unaware of the current refactor.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/536/commits/ad6d4de42589b04634fc98feaabc7b7dba87d8f7.